tests/gce_virtual_machine_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for perfkitbenchmarker.gcp.gce_virtual_machine"""
+
+import unittest
+import mock
+
+from perfkitbenchmarker import pkb  # noqa
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import virtual_machine
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.gcp import gce_virtual_machine
+
+
+class GCEPreemptibleVMFlagTestCase(unittest.TestCase):
+  def testPreemptibleVMFlag(self):
+    with mock.patch(vm_util.__name__ + '.IssueCommand') as issueCommand, \
+            mock.patch('__builtin__.open'), \
```

Comment:
Will creating a temporary file in your test cause issues? If not, might be more realistic to leave them.